### PR TITLE
support listing loadlib and clear pass error

### PR DESCRIPTION
**Pull request by tiantn**
Signed-off-by: tian na <tiantn@cn.ibm.com>

### CODE REVIEW

**Comment by dkelosky** [comment]
It looks like this is targeting #81 and #85.  

#85 seems fixed; here is the new error:
```
Command Error:
Username or password are not valid or expired.
```

File: src/FTPBase.Handler.ts [hunk 1]
```diff
@@ -36,7 +36,12 @@ export abstract class FTPBaseHandler implements ICommandHandler {
             await this.processFTP(additionalParameters);
         } catch (e) {
             this.log.error("Error encountered in FTP command:\n%s", require("util").inspect(e));
+            if (e.message.indexOf("PASS command failed") !== -1) {
```
Comment (dkelosky) [inline src/FTPBase.Handler.ts:39]: these code changes make sense for #85 but I don't see any code changes that address #81.  listing a loadlib data set I still get:
```
Command Error:
Unrecognized file list header:  Name      Size     TTR   Alias-of AC --------- Attributes --------- Amode Rmode
```

Comment (tiantn) [reply src/FTPBase.Handler.ts:39]: Thank you @dkelosky, to support listing loadlib dataset members, the main code change is in zos-node-accessor, and I just update the zos-node-accesor’s version in package.json. Now it is 1.0.9 and you may need run npm install again to update the modules.

**Review by dkelosky — APPROVED** [review]

File: src/FTPBase.Handler.ts (hunk 1 above)
Comment (zFernand0) [reply src/FTPBase.Handler.ts:39]: Shouldn't we make the regular dependencies static?
Meaning no `^` on `package.json#dependencies`
https://github.com/zowe/zowe-cli-ftp-plugin/blob/d7f541a493c24dfd1ea6bf2fbb1acee3608e5224/package.json#L36-L38

Comment (std4lqi) [reply src/FTPBase.Handler.ts:39]: Agree to use static version number in dependencies, to avoid the potential npm subtitution attacks.

**Review by std4lqi — APPROVED** [review]

File: src/FTPBase.Handler.ts (hunk 1 above)
Comment (tiantn) [reply src/FTPBase.Handler.ts:39]: Thanks @zFernand0 @std4lqi ! I will update it. Thank you very much!

**Review by lauren-li — COMMENTED** [review]
@tiantn I think this looks good! I was able to manually list loadlibs, and the zftp error for the incorrect/missing credentials is also much clearer. All unit tests are passing.

The only thing left is a system test for listing loadlibs that keeps failing:
Screen Shot 2021-07-26 at 6 14 28 PM
This might have to do with the way my environment or test profile data is set up. @tiantn if you have any pointers for this, I would greatly appreciate it! Thank you!

**Comment by tiantn** [comment]
Hi @std4lqi, @zFernand0, I updated the zos-node-accessor version to static. Thank you! And @lauren-li , thank you for your test, I found I missed the dsnLoadLib in default_properties. Now I added it and you just need input the loadlib dataset name in the custom_properties to run the system test. Thank you very much!

**Comment by lauren-li** [comment]
> Hi @std4lqi, @zFernand0, I updated the zos-node-accessor version to static. Thank you! And @lauren-li , thank you for your test, I found I missed the dsnLoadLib in default_properties. Now I added it and you just need input the loadlib dataset name in the custom_properties to run the system test. Thank you very much!

@tiantn, I added the new loadlib dataset name into my custom_properties and re-ran the system tests. All looks good now. Thank you for the update!

**Review by lauren-li — APPROVED** [review]

**Comment by zFernand0** [comment]
I didn't get to test the functionality, but the code changes LGTM! 😋

**Pull request merged**